.github/workflows/python-publish.yml

```diff
@@ -33,3 +33,16 @@ jobs:
     - name: Publish package
       if: startsWith(github.ref, 'refs/tags')
       uses: pypa/gh-action-pypi-publish@81e9d935c883d0b210363ab89cf05f3894778450
+    - name: Build documentation
```

Comment:
I don't think we need to build the docs in GitHub Actions. RTD can build the docs with just build instructions in .readthedocs.yaml, which is what the rope repository did.  

If we do have some legitimate reason to build docs here, the same reason probably applies to the rope repository as well.